src/App.js

```diff
@@ -2,10 +2,14 @@ import React from 'react';
 import './App.css';
 
 function App() {
-  return (
-    <div>
+  let handleAlert = (a) => {
```

Comment:
const

Reply:
remove argument

Reply:
done with const and argument, also made some changes to put a button in the centre, but gh-pages hasn't been responding yet. Also could you pleas tell me what do you mean with .gitignore? I used "git add .", should I just use "git add <name_of_the_files>"? Thanks a lot!